apps/web/src/services/administration/components/RoleDeleteModal.vue

```diff
@@ -100,49 +85,17 @@ const handleDelete = async () => {
     }
 };
 
-const projectFieldHandler = (value, projects) => {
-    if (value) {
-        const projectId = value.project_info?.project_id;
-        if (projectId) {
-            return projects[projectId] ? projects[projectId].label : projectId;
-        }
-        const projectGroupId = value.project_group_info?.project_group_id;
-        return state.projectGroups[projectGroupId] ? state.projectGroups[projectGroupId].label : projectGroupId;
-    }
-    return '--';
-};
-
-const getProjectLink = (value) => {
-    const projectId = value?.project_info?.project_id;
-    let link;
-    if (projectId) {
-        link = router.resolve(referenceRouter(projectId, {
-            resource_type: 'identity.Project',
-        }));
-    } else {
-        const projectGroupId = value?.project_group_info?.project_group_id;
-        link = router.resolve(referenceRouter(projectGroupId, {
-            resource_type: 'identity.ProjectGroup',
-        }));
-    }
-    return link.href;
-};
 const getRoleBindingList = () => Promise.all(rolePageStore.selectedRoles.map(async (role) => {
     try {
-        const { results }: ListResponse<RoleBindingModel> = await SpaceConnector.client.identity.roleBinding.list({ role_id: role.role_id });
-        const roleBindingList: UnDeletableRole[] = results.map((roleBinding) => {
-            const {
-                // eslint-disable-next-line @typescript-eslint/naming-convention
-                resource_id, resource_type, role_info, project_info, project_group_info,
-            } = roleBinding;
-            return {
-                roleName: role_info?.name,
-                roleDescription: role_info?.tags?.description,
-                roleType: role_info?.role_type,
-                assignTo: { resource_id, resource_type },
-                project: (project_info || project_group_info) ? { project_info, project_group_info } : undefined,
-            };
+        const { results } = await SpaceConnector.clientV2.identity.roleBinding.list<RoleBindingListParameters, RoleBindingListResponse>({
```

Comment:
Suggestion: I think it can be used like this.

```suggestion
        const { results } = await SpaceConnector.clientV2.identity.roleBinding.list<RoleBindingListParameters, ListResponse<RoleBindingModel>>({
```